### PR TITLE
feat mobile: clan notifications setting

### PR DESCRIPTION
**Pull request by loi-huynhphuc**

### CODE REVIEW

**Comment by loi-huynhphuc** [comment]
https://github.com/user-attachments/assets/f272f6ba-dc71-48b7-b18f-a533edaf97bf

File: apps/mobile/src/app/components/ClanNotificationSetting/CategoryChannel/index.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+				}
+			}
+			return 0;
+		});
+		return settingOptions;
+	}, [channelCategorySettings]);
+
+	return (
+		<Block borderRadius={size.s_14} overflow="hidden">
+			{sortedChannelCategorySettings?.length
```
Comment (phuocnguyenncc) [inline apps/mobile/src/app/components/ClanNotificationSetting/CategoryChannel/index.tsx]: convert sortedChannelCategorySettings?.length to boolean. !!

File: apps/mobile/src/app/components/ClanNotificationSetting/NotificationOverrides/index.tsx [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+							placeholder={'Search'}
+							style={styles.input}
+							value={searchText}
+							onChangeText={onTextChange}
+						/>
+						<Icons.MagnifyingIcon width={20} height={20} color={themeValue.text} />
+					</Block>
+				</Block>
+				<ScrollView>
+					{filteredOptions?.length
```
Comment (phuocnguyenncc) [inline apps/mobile/src/app/components/ClanNotificationSetting/NotificationOverrides/index.tsx]: convert .length to boolean

File: apps/mobile/src/app/screens/home/homedrawer/components/ClanMenu/ClanMenu/index.tsx [hunk 3]
```diff
@@ -43,6 +43,10 @@ export default function ClanMenu({ inviteRef }: IServerMenuProps) {
 		dismiss();
 	};
 
+  const handelOpenNotifications = () => {
+    navigation.navigate(APP_SCREEN.MENU_CLAN.STACK, { screen: APP_SCREEN.MENU_CLAN.NOTIFICATION_SETTING });
+		dismiss();
+  }
```
Comment (phuocnguyenncc) [inline apps/mobile/src/app/screens/home/homedrawer/components/ClanMenu/ClanMenu/index.tsx]: wrapper useCallback

**Review by phuocnguyenncc — APPROVED** [review]

**Pull request merged**